rest/model/task.go

```diff
@@ -275,6 +275,14 @@ func (at *APITask) BuildFromService(t interface{}) error {
 			}
 		}
 
+		isKnown, err := v.IsKnownFailure()
+		if err != nil {
+			return errors.Wrapf(err, "Error checking if the task is a known issue: %v", err.Error())
+		}
+		if isKnown {
+			v.Status = evergreen.TaskKnownIssue
```

Comment:
I'm concerned about making this change here, because BuildFromService is used for API routes as well. I think either this should be done in the resolver, or there should be an additional APITask field called IsKnownIssue or something (then this information could still return whether it's a known issue to API routes without changing what users might currently expect)